.lando.yml

```diff
@@ -5,6 +5,9 @@ config:
   xdebug: false
   php: '7.4'
 services:
+  xdebug: true
```

Comment:
IMHO, by default, it should be disabled.

Reply:
i've gone back and forth on this. it seems like you have to rebuild the container if you want to turn it on if this is set to false, which is annoying (but having it on is also annoying).